ccm_web/package.json

```diff
@@ -19,75 +19,74 @@
   "author": "",
```

Comment:
I see on high severity and it suggest to fix `node_modules/moment` library. It is not CCM direct dependency but some libraries use it.

Reply:
Should be resolved now.